.github/workflows/Test.yml

```diff
@@ -0,0 +1,79 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+
+env:
+  GO111MODULE: on
+
+jobs:
+
+
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+        - { icon: 🐧, os: ubuntu }
+        - { icon: 🍎, os: macOS }
+        go:
+        - 1.14.x
+        - 1.15.x
+    name: '${{ matrix.platform.icon }} ${{ matrix.go }}'
+    runs-on: ${{ matrix.platform.os }}-latest
+    steps:
+
+    - name: '⚙️ Setup go'
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go }}
+
+    - name: '🧰 Checkout'
+      uses: actions/checkout@v2
+
+    - name: '⚙️ Install deps'
+      run: |
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | bash -s -- -b $HOME/go/bin/ latest
+        go install github.com/kyoh86/richgo
+        go install github.com/mitchellh/gox
+
+    - name: '🚧 Run tests'
+      run: PATH=$HOME/go/bin/:$PATH make
```

Comment:
Will this run all the make tasks? Like `test`, `fmt` etc.

Reply:
Yes:

- https://github.com/spf13/cobra/blob/master/Makefile#L16
- https://github.com/spf13/cobra/blob/master/Makefile#L26